### PR TITLE
[DOCS-7627] Add OTel Kafka Metrics Docs

### PR DESCRIPTION
**Pull request by mackjmr**
### What does this PR do? What is the motivation?
Add OTel Kafka Metrics Docs.

### CODE REVIEW

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+      - topics
+      - consumers
+```
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+The kafka metrics receiver needs to be used in a collector in `deployment` mode to avoid collecting the same metrics multiple times.
+
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/kafka_metrics.md:44]: Can we add diagram -here . How to run in Gateway deployment ?

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+# JMX Metrics Gatherer Jar
+ARG JMX_GATHERER_JAR_VERSION=1.27.0
+ADD https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${JMX_GATHERER_JAR_VERSION}/opentelemetry-jmx-metrics.jar /opt/opentelemetry-jmx-metrics.jar
+# nonroot user id (https://groups.google.com/g/distroless-users/c/-DpzCr7xRDY/m/eQqJmJroCgAJ)
+ARG USER_UID=65532
+RUN chown ${USER_UID} /opt/opentelemetry-jmx-metrics.jar
+
+
+FROM gcr.io/distroless/java11-debian11:nonroot
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/kafka_metrics.md:126]: use java 17 ?

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+        action: insert
+```
+
+
+## Data collected
+
+| OpenTelemetry Metric | DD Metric | Source | Transform done DD Metric |
+|---|---|---|---|
+| otel.kafka.producer.request-rate | kafka.producer.request_rate | JMX Receiver / JMX Metrics Gatherer {target_system:kafka-producer} | |
+| otel.kafka.producer.response-rate | kafka.producer.response_rate | JMX Receiver / JMX Metrics Gatherer {target_system:kafka-producer} | |
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/kafka_metrics.md:260]: Should we move this to metrics mapping page?

**Comment by buraizu** [comment]
Added the `Work in Progress` label for now to remove from the documentation team's review queue, as there is feedback that needs to be resolved before we can properly review. Please remove the label once it's ready for review, or feel free to reach out on the documentation channel.

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+<div class="alert alert-warning">
+OTel Kafka Metrics Remapping is in public alpha. It is available in versions >= 0.93.0 of the collector. If you have feedback related to this, reach out your account team to provide your input.
+</div>
+
+
+## Overview
+
+{{< img src="/opentelemetry/collector_exporter/kafka_metrics.png" alt="OpenTelemetry Kafka metrics in OOTB Kafka dashboard" style="width:100%;" >}}
+
+The [kafka metrics receiver][1], [JMX Receiver][2]/ [JMX Metrics Gatherer][3] allow collecting kafka metrics and access to the OOTB Kafka Dashboard "Kafka, Zookeeper and Kafka Consumer Overview".
```
[the remark this answers is not in the document]
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/kafka_metrics.md]: expanded and added link 👍

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+```yaml
+processors:
+  attributes:
+    actions:
+      - key: ddtags
+        value: "source:kafka"
+        action: insert
+```
+
```
[the remark this answers is not in the document]
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/kafka_metrics.md]: Added explanation on how to ensure this.

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md (hunk 2 above)
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/kafka_metrics.md]: Updated.

File: content/en/opentelemetry/collector_exporter/kafka_metrics.md (hunk 3 above)
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/kafka_metrics.md]: Moved.

**Review by dineshg13 — APPROVED** [review]

**Comment by mackjmr** [comment]
@buraizu this should be ready for review now, removed the work in progress label 👍

**Comment by buraizu** [comment]
Thanks, created DOCS-7627 to review

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+<div class="alert alert-warning">
+OTel Kafka Metrics Remapping is in public alpha. It is available in versions >= 0.93.0 of the collector. If you have feedback related to this, reach out your account team to provide your input.
+</div>
+
+
+## Overview
+
+{{< img src="/opentelemetry/collector_exporter/kafka_metrics.png" alt="OpenTelemetry Kafka metrics in OOTB Kafka dashboard" style="width:100%;" >}}
+
+The [kafka metrics receiver][1], [JMX Receiver][2]/ [JMX Metrics Gatherer][3] allow collecting kafka metrics and access to the out of the box [Kafka Dashboard][7] "Kafka, Zookeeper and Kafka Consumer Overview".
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:18]:
```suggestion
The [Kafka metrics receiver][1], [JMX Receiver][2]/ [JMX Metrics Gatherer][3] allow collecting Kafka metrics and access to the out of the box [Kafka Dashboard][7], "Kafka, Zookeeper and Kafka Consumer Overview". 
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+```
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## Log Collection
+
+See [Log Collection][4] for instructions on how to collect logs using the OpenTelemetry Collector.
+
+In order to appear in the OOTB Kafka Dashboard, the kafka logs need to be tagged with `source:kafka`. In order to do so, you can use an attributes processor:
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:243]:
```suggestion
To appear in the out-of-the-box Kafka Dashboard, the Kafka logs need to be tagged with `source:kafka`. To do this, use an attributes processor:
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+```yaml
+processors:
+  attributes:
+    actions:
+      - key: ddtags
+        value: "source:kafka"
+        action: insert
+```
+
+In order to ensure this attribute only gets added to your kafka logs, you can use [include/exclude filtering][8] of the attributes processor.
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:254]:
```suggestion
In order to ensure this attribute only gets added to your Kafka logs, use [include/exclude filtering][8] of the attributes processor.
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+      - brokers
+      - topics
+      - consumers
+```
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## JMX Receiver
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:67]:
```suggestion
## JMX receiver
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+      - brokers
+      - topics
+      - consumers
+```
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+The kafka metrics receiver needs to be used in a collector in `deployment` mode with a single replica. This ensures that the same metric is not collected multiple times. The collector in deployment mode can then leverage the Datadog Exporter to export the metrics directly to Datadog, or leverage the OTLP exporter to forward the metrics to another collector instance.
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:43]:
```suggestion
The Kafka metrics receiver needs to be used in a collector in `deployment` mode with a single replica. This ensures that the same metric is not collected multiple times. The collector in deployment mode can then leverage the Datadog Exporter to export the metrics directly to Datadog, or leverage the OTLP exporter to forward the metrics to another collector instance. 
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+
+{{< /tabs >}}
+
+## JMX Receiver
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+The JMX Receiver has the following requirements:
+- JRE is available on the host in which you are running the collector.
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:73]:
```suggestion
- JRE is available on the host where you are running the collector.
```

File: content/en/opentelemetry/integrations/kafka_metrics.md [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+"-Dotel.metrics.exporter=otlp", \
+"-Dotel.exporter.otlp.endpoint=http://otelcol:4317", \
+"-jar", \
+"/opt/opentelemetry-jmx-metrics.jar"]
+```
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## Log Collection
```
Comment (brett0000FF) [inline content/en/opentelemetry/integrations/kafka_metrics.md:239]:
```suggestion
## Log collection
```

**Review by brett0000FF — COMMENTED** [review]
Looks good! Thanks for writing this up. I left several suggestions to align with our style guide. Please let me know once you've addressed these, and I can approve/merge this. 👍

**Comment by mackjmr** [comment]
Thanks @brett0000FF, applied all suggestions 👍

**Review by brett0000FF — APPROVED** [review]
@mackjmr  Great, thanks! Are you ready for me to merge this? It looks good to me now.

**Comment by mackjmr** [comment]
@brett0000FF yes we are good to merge 👍

**Pull request merged**